capi/geos_c.h.in

```diff
@@ -3635,6 +3658,69 @@ extern GEOSGeometry GEOS_DLL *GEOSOffsetCurve(const GEOSGeometry* g,
 
 ///@}
 
+
+/* ====================================================================== */
+/** @name Coverages
+* Functions to work with coverages represented by lists of polygons
+* that exactly share edge geometry.
+*/
+///@{
+
+/**
+* Analyze a coverage (represented as a collection of polygonal geometry
```

Comment:
is this strictly limited to a collection of Polygons, or are MultiPolygons (e.g., with individual polygons may share edges with other MultiPolygons in the collection) also allowed?

Reply:
MultiPolygons are also allowed as coverage elements.